### DISCUSSION TITLE
Has actor.onMicrostep() been implemented yet?

### DISCUSSION THREAD

**Discussion by phshah95** [statelyai/xstate#4156] [General] [answered]
Hello! I am desperately hoping that the feature mentioned in this [discussion](https://github.com/statelyai/xstate/issues/1767) has been implemented somewhere in v5! Or if anyone has any suggestions on how to work around. I saw someone mention making the "always" an "after: { 1: ...}" would trigger the onTransition listener but I would like to avoid doing that if possible.

Thanks!

**Comment by davidkpiano** [comment, marked as answer]
This will be part of an inspection API instead of a specific `onMicrostep` listener. Still a WIP

**Reply by phshah95** [reply]
gotcha thanks. is there any workaround you would suggest for now?

**Reply by phshah95** [reply]
for more context: I am trying to accumulate statemeta across a series of eventless transitions
